core/trino-main/src/main/java/io/trino/operator/aggregation/AggregationHeader.java

```diff
@@ -20,14 +20,21 @@
 public class AggregationHeader
 {
     private final String name;
+    private final String resolvedName;
     private final Optional<String> description;
     private final boolean decomposable;
     private final boolean orderSensitive;
     private final boolean hidden;
 
     public AggregationHeader(String name, Optional<String> description, boolean decomposable, boolean orderSensitive, boolean hidden)
```

Comment:
Can we modify `AggregationFromAnnotationsParser#parseHeader` to use the new constructor with the resolved name ?. And we can remove the old one